### ISSUE TITLE
Split package warning

### ISSUE THREAD

**Issue by kdubois** [apache/camel-quarkus#4330]
Hi, I've been noticing a split package warning when building my camel quarkus projects:

```
[WARNING] [io.quarkus.arc.deployment.SplitPackageProcessor] Detected a split package usage which is considered a bad practice and should be avoided. Following packages were detected in multiple archives: 
- "org.apache.camel.vault" found in [org.apache.camel:camel-api::jar, org.apache.camel:camel-main::jar]
```

(You can reproduce by building this project: https://github.com:kdubois/CamelQuarkusVoter )

**Comment by davsclaus** [comment]
Yeah I think its fixed in Camel 3.20 - As I think that split package is from that version

**Comment by kdubois** [comment]
Ah ok, I'll patiently wait for it to resolve itself then :).  I presume Camel Quarkus 2.15 will be based on Camel 3.20 then?

**Comment by nchudasmasumo** [comment]
Is there any workaround? I am trying with some POC and stuck with the error 

`{"level":"info","ts":1670606733.9583514,"logger":"camel-k.maven.build","msg":"--- quarkus-maven-plugin:2.13.4.Final:build (default) @ camel-k-integration ---"}
{"level":"info","ts":1670606743.5989583,"logger":"camel-k.maven.build","msg":"[io.quarkus.arc.deployment.SplitPackageProcessor] Detected a split package usage which is considered a bad practice and should be avoided. Following packages were detected in multiple archives: \n- \"io.fabric8.openshift.api.model\" found in [io.fabric8:openshift-model-operator::jar, io.fabric8:openshift-model::jar]"}
{"level":"info","ts":1670606744.916758,"logger":"camel-k.maven.build","msg":"------------------------------------------------------------------------"}
{"level":"info","ts":1670606744.9168925,"logger":"camel-k.maven.build","msg":"BUILD FAILURE"}
{"level":"info","ts":1670606744.9169307,"logger":"camel-k.maven.build","msg":"------------------------------------------------------------------------"}
{"level":"info","ts":1670606744.917022,"logger":"camel-k.maven.build","msg":"Total time:  22.692 s (Wall Clock)"}
{"level":"info","ts":1670606744.917071,"logger":"camel-k.maven.build","msg":"Finished at: 2022-12-09T17:25:44Z"}
{"level":"info","ts":1670606744.9171903,"logger":"camel-k.maven.build","msg":"------------------------------------------------------------------------"}
{"level":"error","ts":1670606744.9271736,"logger":"camel-k.maven.build","msg":"Failed to execute goal io.quarkus:quarkus-maven-plugin:2.13.4.Final:build (default) on project camel-k-integration: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors","stacktrace":"github.com/apache/camel-k/pkg/util/log.Logger.Error\n\tgithub.com/apache/camel-k/pkg/util/log/log.go:75\ngithub.com/apache/camel-k/pkg/util/maven.normalizeLog\n\tgithub.com/apache/camel-k/pkg/util/maven/maven_log.go:83\ngithub.com/apache/camel-k/pkg/util/maven.mavenLogHandler\n\tgithub.com/apache/camel-k/pkg/util/maven/maven_log.go:54\ngithub.com/apache/camel-k/pkg/util.scan\n\tgithub.com/apache/camel-k/pkg/util/command.go:77\ngithub.com/apache/camel-k/pkg/util.RunAndLog.func1\n\tgithub.com/apache/camel-k/pkg/util/command.go:56\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\tgolang.org/x/sync@v0.0.0-20220601150217-0de741cfad7f/errgroup/errgroup.go:75"}
`

**Comment by ppalaga** [comment]
@nchudasmasumo I do not think the warning can cause your build failure. Please consider filing a separate issue with proper steps to reproduce at [Camel K ](https://github.com/apache/camel-k/issues)

**Comment by jamesnetherton** [comment]
The warning is not present with Camel 3.20.x. It'll be fixed in the Camel Quarkus 2.16.0 release.
